.travis.yml

```diff
@@ -59,7 +59,7 @@ script:
   # Test ruby gem
   - gem install gem-path --no-ri --no-rdoc
   - cd $HOME/gems/symengine-0.1.0/
-  - bundle exec rspec
+  - bundle exec rspec --require ./spec/spec_helper.rb
```

Comment:
I'm usually including it :)
And, if we'll look at popular gems, they are typically doing the same thing.
So, I think it is ok (as well as `.rspec` file inclusion).